src/Stores/User/reducer.js

```diff
@@ -0,0 +1,25 @@
+/* @flow */
+import type { User, Action } from '../../types';
+import { ActionTypes } from './actions';
+export type State = {
+  +data: ?User,
+};
+
+const initialState = {
+  data: null,
```

Comment:
set error and loading into initialState?

Reply:
good catch